lib/cli/src/commands/create_exe.rs

```diff
@@ -1357,30 +1362,36 @@ impl LinkCode {
 #[cfg(feature = "http")]
 mod http_fetch {
     use anyhow::{anyhow, Context, Result};
-    use http_req::{
-        request::Request,
-        response::{Response, StatusCode},
-        uri::Uri,
-    };
+    use http_req::{request::Request, response::StatusCode, uri::Uri};
     use std::convert::TryFrom;
+    use target_lexicon::OperatingSystem;
 
     pub fn get_latest_release() -> Result<serde_json::Value> {
         let mut writer = Vec::new();
         let uri = Uri::try_from("https://api.github.com/repos/wasmerio/wasmer/releases").unwrap();
 
-        let response = Request::new(&uri)
-            .header("User-Agent", "wasmer")
+        // Increases rate-limiting in GitHub CI
+        let auth = std::env::var("GITHUB_TOKEN");
+        let mut response = Request::new(&uri);
+
+        if let Ok(token) = auth {
+            response.header("Authorization", &format!("Bearer {token}"));
+        }
+
+        let response = response
+            .header("User-Agent", "wasmerio")
             .header("Accept", "application/vnd.github.v3+json")
             .timeout(Some(std::time::Duration::new(30, 0)))
             .send(&mut writer)
             .map_err(anyhow::Error::new)
             .context("Could not lookup wasmer repository on Github.")?;
 
         if response.status_code() != StatusCode::new(200) {
-            return Err(anyhow!(
-                "Github API replied with non-200 status code: {}",
-                response.status_code()
-            ));
+            eprintln!(
+                "Warning: Github API replied with non-200 status code: {}. Response: {}",
+                response.status_code(),
+                String::from_utf8_lossy(&writer),
+            );
```

Comment:
It might be better to use `log::warn!()` here. That way the warning can go into our logging system, which might end up being saved to a log file or sent to analytics.